minor corrections

SWCCGArenaProject opened the pull request:
I write software that uses this json data alongside the gemp swccg server like Epic Duel and Rando Cal (the swccg playing bot). That automation does a deep comparison between the gemp source code and the json project to make sure things match in all fields that exist in both projects. I found 3 minor issues that prevent matching json cards to gemp cards:

"•Grotto Werribee" has a power of 2 in the json data, but a power of 1 in the gemp source code. I checked the printed card and the power should be 1.
"•Gardulla The Hutt (v)" this dark legacy card is using a lower case "v" - not a major problem but all the other cards use uppercase "V".
"Dorme'" the accent mark is missing in the gemp source code, that team did not want it added there so they suggested removing it here. I'd be fine reverting this part since it's just one card I can easily write a rule for it in the card matcher.

Gergall reviewed (changes requested):
Love the Grotto fix and the Gardulla fix.

The Printings fix for Rapid Deployment appears you may have been looking at the wrong part of the file.  You've mistakenly added the printing to Tunnel Vision!  And in fact if you look at Rapid Deployment, it already has a 107 printing listed.  In short, it appears that no Light.json changes are needed.

SWCCGArenaProject commented:
> Love the Grotto fix and the Gardulla fix.
> 
> The Printings fix for Rapid Deployment appears you may have been looking at the wrong part of the file. You've mistakenly added the printing to Tunnel Vision! And in fact if you look at Rapid Deployment, it already has a 107 printing listed. In short, it appears that no Light.json changes are needed.

Thank you, and yeah you are 100% - my automation had a bug that made me this Rapid Deployment was wrong but it never was, I reverted that and the weird line ending changes I didn't mean to do those either.

I've added a fix for "Dorme'" to remove the accent mark, that gemp team suggested that but I can easily code around that if you don't want that change in the json.
